README.md

```diff
@@ -38,6 +38,18 @@ FLAGS:
 
 OPTIONS:
         --commit-sha <COMMIT HASH>                   Sets the hash of the commit used to generate the code coverage data
+        --excl-br-line <regex>
+            Lines in covered files containing this marker will be excluded.
+        --excl-br-start <regex>
+            Marks the beginning of an excluded section. The current line is part of this section.
+        --excl-br-stop <regex>
+            Marks the end of an excluded section. The current line is part of this section.
```

Comment:
... `Marks the end of a section excluded from branch coverage. The current line is part of this section.` ?